src/utils/api.ts

```diff
@@ -115,6 +118,25 @@ export class API {
     }
   }
 
+  async getDeployments(
+    id: string,
```

Comment:
Since there are several IDs we are dealing with, it would be great to give it a more specific name, e.g. `projectId`